pages/api/graphql.js

```diff
@@ -1,7 +1,13 @@
 import { ApolloServer } from 'apollo-server-micro';
 import { schema } from '@/utils/apollo/schema';
+import dbConnect from '@/utils/dbConnect';
 
-const apolloServer = new ApolloServer({ schema });
+const apolloServer = new ApolloServer({
+  schema,
```

Comment:
We can pass in our database connection into the server, but I don't think we even need to connect to database at all in resolvers.js